src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/DocumentsViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package org.vaccineimpact.orderlyweb.viewmodels
+
+import org.vaccineimpact.orderlyweb.ActionContext
+import org.vaccineimpact.orderlyweb.db.AppConfig
+import org.vaccineimpact.orderlyweb.models.Document
+
+data class DocumentsViewModel(val docs: List<Document>, val appViewModel: AppViewModel) : AppViewModel by appViewModel
+{
+    companion object
+    {
+        val breadcrumb = Breadcrumb("Docs", "${AppConfig()["app.url"]}/docs")
```

Comment:
Have we got a more offical name than 'Docs'?

Reply:
Lol, not sure. I'll ask Kim!

Reply:
Just need to update this Breadcrumb for the new project-docs route..